.github/workflows/publish.yml

```diff
@@ -24,18 +55,16 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.12'
-      - name: Install dependencies
+          python-version: 3.9
+      - name: Install flit
         run: |
-          python -m pip install --upgrade pip
-          pip install build
-      - name: Build package
-        run: python -m build
-      - name: Publish package
-        uses: pypa/gh-action-pypi-publish@81e9d935c883d0b210363ab89cf05f3894778450
-        with:
-          user: __token__
-          password: ${{ secrets.PYPI_API_TOKEN }}
+          pip install flit
+      - name: Publish the distibution to PyPI
+        run: flit publish
+        env:
+          FLIT_INDEX_URL: https://upload.pypi.org/legacy/
```

Comment:
For my education, are we using the legacy endpoint because we're not using OIDC?

Reply:
The term legacy here does not mean this is deprecated or outdated, the legacy endpoint is recommended for uploads with twine or flit while the 'new' endpoint is recommended for web uploads https://warehouse.pypa.io/api-reference/legacy.html#upload-api